kiwi/system/setup.py

```diff
@@ -890,3 +910,15 @@ def _export_deb_package_verification(self, filename):
         )
         with open(filename, 'w') as verified:
             verified.write(query_call.output)
+
+    def _systemd_firstboot_has_option(self, flag):
```

Comment:
I agree we need this option check to see the capability of a command. I also fear we might need this type of check more often in the future and as I know you and me are not a friend of it, I still think we should put this code into an extra class so that we can have it available in all parts of kiwi. What do you think. I suggest to put that into utils/command_capabilities.py and maintain it as class method so that you can write e.g

```python
CommandCapabilities.has_option_in_help(call='systemd-firstboot' root='...')
```

Help lookup is pretty common and should work the same way for almost all commands. Maybe more sophisticated checks are needed in the future and those could go there and are collective all at the same place

What do you think ?

A little more effort to start with but I think it's worth it

Reply:
Sure you are I right,  I buy the idea, it makes sense to me.

Reply:
ok great that means you plan to add commits to this one before we merge or want to do it in an extra pull request ?

Reply:
I will add commits here, I was thinking on making the deprecated suseConfig an extra PR, here I will just add the CommandCapabilities class

Reply:
👍